src/foram3d/simulation.ts

```diff
@@ -49,6 +53,9 @@ module Foram3D {
       this.reset();
 
       this.foram = new Foram(genotype, numChambers);
+
+      this.targetControls.fitTarget(this.foram);
```

Comment:
`this.fitTarget()`

Nie ma się co powtarzać. Jeden dodatkowy `if` nas nie pokrąży myślę :)